pinot-common/src/main/java/org/apache/pinot/common/config/provider/TableCache.java

```diff
@@ -232,7 +232,11 @@ public Map<String, String> getColumnNameMap(String rawTableName) {
   @Nullable
   public Map<Expression, Expression> getExpressionOverrideMap(String tableNameWithType) {
```

Comment:
This was brought up in another PR as well. `tableNameWithType` is misleading. Rename to `physicalOrLogicalTableName` ?